zzzz_modular_occulus/code/game/objects/items/devices/traitor_devices.dm

```diff
@@ -10,4 +10,296 @@
 
 	/* Changes from original file: origin_tech was listed as TECH_MAGNETS,
 	   which was a non-existent tech that broke the destructive analyzer.
-	*/
\ No newline at end of file
+	*/
+var/global/list/active_radio_jammers = list()
+
+/proc/is_jammed(var/obj/radio)
```

Comment:
Not answered.

Reply:
Because thats how it was in the code I ported, and it works?

Reply:
I believe it is so you can jam mobs as well as objects. Considering this is a port, it may be best to leave it as-is unless it's causing an issue with conflicts.